doc/source/whatsnew/v1.6.0.rst

```diff
@@ -220,7 +220,7 @@ Indexing
 Missing
 ^^^^^^^
 - Bug in :meth:`Index.equals` raising ``TypeError`` when :class:`Index` consists of tuples that contain ``NA`` (:issue:`48446`)
--
+- Bug in :meth:`Series.map` resulting in incorrect mapping when NaNs are present (:issue:`48813`)
```

Comment:
Nit: Could you mention specifically when a defaultdict was provided?

Reply:
updated 👍